### ISSUE TITLE
Allow ipydeps to operate without pypki2

### ISSUE THREAD

**Issue by somedave** [nbgallery/ipydeps#12]
It'd be nice to run ipydeps in instances where pypki2 is not needed.  Currently if you try to import ipydeps, it tries to launch pypki2 and gives you this error:

```
Javascript error adding output!
ReferenceError: MyPKI is not defined
See your browser Javascript console for more details.
```

**Comment by billallen256** [comment]
This makes sense as it should really be the default behaviour to not need pypki2.  Would adding a ```--with-pypki2``` option to ipydeps.conf be sufficient?

**Comment by somedave** [comment]
Yea that sounds good @gershwinlabs

**Comment by somedave** [comment]
hey @gershwinlabs any chance you can take a quick crack at this?  I think what you suggested above would do the trick.

**Comment by billallen256** [comment]
@somedave Sorry for the delay.  Growing family has a way of putting significant limits on available time.  Anyway, I think I've made the necessary changes.  It's in the optional_pypki2 branch.  The unit tests passed, but I haven't had a chance to do any integration testing.  Can you guys take it for a spin?

**Comment by jeffyjefflabs** [comment]
I think when `--with-pypki2` is set, you also want to import pypki2.pipwrapper, yes?

**Comment by billallen256** [comment]
Fixed in a863c1a67cf1a110a1d5df49b8f8cf45972cdf4f
